src/core/providers/selectProvider.js

```
@@ -0,0 +1,118 @@
+      reason: 'local-worker-unavailable',
+      fallback: true,
+      caps,
+      candidates: { onnx, browserSam, worker },
+    };
+  }
+
+  return { provider: onnx, reason: 'unknown-mode', candidates: { onnx, browserSam, worker } };
+}
+
+function readEnvMode() {
+  if (typeof process !== 'undefined' && process.env && process.env.SAM_AUDIO_MODE) {
```

ESLint

[error] 'process' is not defined.

(no-undef)